BH/Modules/Item/ItemDisplay.cpp

```diff
@@ -1095,6 +1216,28 @@ bool ResistAllCondition::EvaluateInternalFromPacket(ItemInfo *info, Condition *a
 			IntegerCompare(pRes, operation, targetStat));
 }
 
+bool AddCondition::EvaluateInternal(UnitItemInfo *uInfo, Condition *arg1, Condition *arg2) {
+	int value = 0;
+	vector<string> codes = split(key, '+');
```

Comment:
Please don't do string splits/compares in an evaluation method - this gets called *a lot*. Do any heavy lifting/string parsing etc when you create the `AddCondition` object.